src/plugins/intel_cpu/src/nodes/executors/interpolate.cpp

```diff
@@ -431,7 +431,7 @@ void ov::intel_cpu::InterpolateExecutor::buildTblCubic(const VectorDims& srcDimP
     int tblAdvance = 0;
     int* xOrigin = static_cast<int*>(&indexTable[tblAdvance]);
     tblAdvance += OW;
-    float* xFactor = reinterpret_cast<float*>(&indexTable[tblAdvance]);
+    auto* xFactor = reinterpret_cast<float*>(&indexTable[tblAdvance]);
```

Comment:
This looks very inconsistent.
Let's use `MinTypeNameLength=3` to at least cover all the fundamental types.

Reply:
Done